flocker/dockerplugin/test/test_api.py

```diff
@@ -112,13 +122,39 @@ def create_after_list():
             return d
         self.flocker_client.list_datasets_configuration = create_after_list
 
-        return self.assertResult(
-            b"POST", b"/VolumeDriver.Create",
-            {u"Name": name}, OK, {u"Err": None})
+        return self.create(name)
+
+    def test_mount(self):
+        """
+        ``/VolumeDriver.Mount`` sets the primary of the dataset with matching
+        name to the current node and then waits for the dataset to
+        actually arrive.
+        """
+        name = u"myvol"
+        dataset_id = UUID(dataset_id_from_name(name))
+        # Create dataset on a different node:
+        d = self.flocker_client.create_dataset(
+            self.NODE_B, DEFAULT_SIZE, metadata={u"name": name},
+            dataset_id=dataset_id)
+
+        # In 0.1 seconds the dataset arrives as state:
+        reactor.callLater(0.1, self.flocker_client.synchronize_state)
```

Comment:
Is this `0.1` deliberately chosen to be clearly greater than `0.05`? If so, then please make the `0.05` a named constant and make this `2 * self.flocker_client._POLLING_INTERVAL`.